release-notes.txt

```diff
@@ -0,0 +1,208 @@
+BISDN Linux
+v4.6.0 Release Notes
+Date: 2022-05-03
+
+! New in this release
+- New platform Accton-AS5835-54x is now supported
+- baseboxd: bump version to 1.11.2
+- Licenses in build process were updated to use the correct SPDX identifier
+- Linux kernel updated to version 5.15.35
+- OF-DPA updated to support configuraton of MPLS pseudowires using RYU
```

Comment:
is this the correct way? AFAIU Ryu now supports configuring MPLS pseudowires in OF-DPA, not the other way around